robotpy_installer/deploy.py

```diff
@@ -169,6 +175,28 @@ def run(self, options, robot_class, **static_options):
         if not hostname_or_team and options.team:
             hostname_or_team = options.team
 
+        if options.pre_deploy_script:
+            import os.path
+            predeploy = os.path.abspath(options.pre_deploy_script)
+            if not os.path.exists(predeploy):
+                print_err(f"ERROR: Pre-deploy script {repr(predeploy)} not found")
+                return 1
+            print_err(f"Running pre-deploy script {repr(predeploy)}")
+            try:
+                # The pre-deploy script will be executed with the user's
+                # permissions.  We assume this is not a security issue,
+                # since the user is explicitly choosing on the command line
+                # to run the script.  If a pre-deploy script could be run
+                # automatically (based on its name, e.g.), there might be
+                # security concerns, especially when checking out an
+                # external repo to test someone else's code.
+                #
+                args = ['python3', predeploy, '--robot_path', robot_path]
+                subprocess.check_output(args)
```

Comment:
You aren't using the output of the script, so you'd want to use `subprocess.run(... check=True)` instead.